persistence/rest/src/main/java/org/infinispan/persistence/rest/RestStore.java

```diff
@@ -265,7 +260,7 @@ private String getHeader(String name, RestResponse restResponse) {
       } catch (Exception e) {
          throw new PersistenceException(e);
       } finally {
-         if(response != null) {
+         if (response != null) {
```

Comment:
Since a RestResponse is AutoCloseable, you could probably convert this to a try-with-resources

Reply:
Done